dvc/stage/imports.py

```diff
@@ -1,9 +1,16 @@
 import logging
 
+from dvc.exceptions import InvalidArgumentError
+
 logger = logging.getLogger(__name__)
 
 
 def _update_import_on_remote(stage, remote, jobs):
+    if stage.is_repo_import:
+        raise InvalidArgumentError(
+            "Can't update a repo import with --to-remote"
+        )
```

Comment:
What's a repo import?

Reply:
BTW I'm not sure that term will mean anything for users. If that exception is a user message I would rephrase to something more explicit like "Data imported with `dvc import` can't be updated `--to-remote`." or "Only data imported with `dvc import-url --to-remote` can be updated `--to-remote`."